<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StyleManager</class>
 <widget class="QWidget" name="StyleManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>419</width>
    <height>318</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>1</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0" colspan="2">
        <widget class="QTabWidget" name="tabs">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>99</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <widget class="QListView" name="paragraphStylesListView">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>14</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <attribute name="title">
           <string>Paragraph</string>
          </attribute>
         </widget>
         <widget class="QListView" name="characterStylesListView">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>14</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <attribute name="title">
           <string>Character</string>
          </attribute>
         </widget>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="bNew">
         <property name="text">
          <string>&amp;New</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="bDelete">
         <property name="text">
          <string>Delete</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="StyleManagerWelcome" name="welcomePage"/>
      <widget class="ParagraphGeneral" name="paragraphStylePage"/>
      <widget class="CharacterGeneral" name="characterStylePage"/>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ParagraphGeneral</class>
   <extends>QWidget</extends>
   <header>dialogs/ParagraphGeneral.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CharacterGeneral</class>
   <extends>QWidget</extends>
   <header>dialogs/CharacterGeneral.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>StyleManagerWelcome</class>
   <extends>QWidget</extends>
   <header>dialogs/StyleManagerWelcome.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
